docs/Process_CI.md

```
@@ -17,61 +20,91 @@ bananapim7/archive/Armbian_[0-9].*Bananapim7_noble_vendor_[0-9]*.[0-9]*.[0-9]*_g
 
+Wildcards apply to all branches:
 ```
-khadas-edge2,legacy:vendor:,ENABLE_EXTENSIONS="image-output-oowow,v4l2loopback-dkms,mesa-vpu"
+BOARD_NAME::ENABLE_EXTENSIONS="ext1,ext2"
 ```
-### Kernel Descriptions for Download Pages?
 
-Each kernel branch can include an optional description, stored in [`kernel-description.json`](https://github.com/armbian/os/blob/main/kernel-description.json).
+???+ Note
+    Manual extensions are **merged** with automatic extensions, not replaced.
+### Kernel Descriptions for Download Pages
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)